src/Cli/Microsoft.TemplateEngine.Cli/LocalizableStrings.resx

```diff
@@ -393,6 +393,9 @@ Run 'dotnet {1} --show-aliases' with no args to show all aliases.</value>
   <data name="ColumnNameAuthor" xml:space="preserve">
     <value>Author</value>
   </data>
+  <data name="ColumnNamePackageNameAndOwners" xml:space="preserve">
+    <value>Package Name / Owners</value>
```

Comment:
please consider adding comments for translators

Reply:
done